src/tts_daemon.py

```
@@ -25,25 +25,150 @@
+    """Create or tighten a private directory used by the daemon."""
+    directory.mkdir(parents=True, exist_ok=True, mode=mode)
+    os.chmod(directory, mode)
+    return directory
+
+
+def ensure_bearer_token_file(token_file: Path) -> str:
+    """Create or load the bearer token from a strict 0600 file."""
+    ensure_private_directory(token_file.parent, 0o700)
+
+    if token_file.is_symlink():
+        raise ValueError(f"Auth token file must not be a symlink: {token_file}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)